### PR TITLE
✨ 2024 -> 2025 + Ben

### PR DESCRIPTION
**Pull request by ctcpip**

### CODE REVIEW

File: spec/collator.html [hunk 1]
```diff
@@ -212,7 +212,7 @@ <h1>
...
             The returned Number represents the result of an implementation-defined locale-sensitive String comparison of _x_ with _y_.
             The result is intended to correspond with a sort order of String values according to the effective locale and collation options of _collator_, and will be negative when _x_ is ordered before _y_, positive when _x_ is ordered after _y_, and zero in all other cases (representing no relative ordering between _x_ and _y_).
-            String values must be interpreted as UTF-16 code unit sequences as described in es2024, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and a surrogate pair (a code unit in the range 0xD800 to 0xDBFF followed by a code unit in the range 0xDC00 to 0xDFFF) within a string must be interpreted as the corresponding code point.
+            String values must be interpreted as UTF-16 code unit sequences as described in es2025, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and a surrogate pair (a code unit in the range 0xD800 to 0xDBFF followed by a code unit in the range 0xDC00 to 0xDFFF) within a string must be interpreted as the corresponding code point.
```
Comment (ljharb) [inline spec/collator.html:215]:
```suggestion
            String values must be interpreted as UTF-16 code unit sequences as described in ES2025, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and a surrogate pair (a code unit in the range 0xD800 to 0xDBFF followed by a code unit in the range 0xDC00 to 0xDFFF) within a string must be interpreted as the corresponding code point.
```

File: spec/conventions.html [hunk 2]
```diff
@@ -2,14 +2,14 @@
   <h1>Notational Conventions</h1>
 
   <p>
-    This standard uses a subset of the notational conventions of the ECMAScript 2024 Language Specification (ECMA-262 15<sup>th</sup> Edition), as es2024:
+    This standard uses a subset of the notational conventions of the ECMAScript 2025 Language Specification (ECMA-262 16<sup>th</sup> Edition), as es2025:
```
Comment (ljharb) [inline spec/conventions.html:5]:
```suggestion
    This standard uses a subset of the notational conventions of the ECMAScript 2025 Language Specification (ECMA-262 16<sup>th</sup> Edition), as ES2025:
```

File: spec/collator.html (hunk 1 above)
Comment (gibson042) [reply spec/collator.html:215]: No, this is correct as-is: https://tc39.es/ecma402/#normative-references
> Throughout this document, the phrase “es2024, _x_” (where x is a sequence of numbers separated by periods) may be used as shorthand for "ECMAScript 2024 Language Specification (ECMA-262 15th Edition, sub clause _x_)". Where _x_ is followed by more such sequences of period-separated numbers, separated from each other by commas, each such sequence is also a shorthand for the corresponding sub clause of ECMA-262.

File: spec/conventions.html (hunk 2 above)
Comment (gibson042) [reply spec/conventions.html:5]: https://tc39.es/ecma402/#normative-references

**Review by gibson042 — APPROVED** [review]

File: spec/collator.html (hunk 1 above)
Comment (ljharb) [reply spec/collator.html:215]: oof, ok - correct per that definition, but not the way it’s usually referred to outside this document.

File: spec/metablock.html [hunk 3]
```diff
@@ -15,6 +15,7 @@ <h1>Contributing to this Specification</h1>
     </li>
     <li>Editors:
       <ul>
+        <li><a href="mailto:benallen@igalia.com">Ben Allen</a> (<a href="https://github.com/ben-allen">@ben-allen</a>)</li>
```
Comment (ctcpip) [inline spec/metablock.html:18]: @ben-allen 👀 checking if this is the email that should appear here

**Pull request merged**